src/linagora.esn.videoconference.calendar/app/components/calendar-videoconference-form/calendar-videoconference.controller.js

```diff
@@ -61,4 +62,33 @@ function calendarVideoconferenceFormController(uuid4, VideoConfConfigurationServ
       self._videoconference !== EVENT_VIDEOCONFERENCE_OPTIONS.NO_VIDEOCONFERENCE) ?
       self.videoConfHostname + self._roomName : undefined;
   }
+
+  function copyVideoConferenceLink() {
+    /**
+     * document.execCommand is currently the only method to copy to clipboard that supports all browsers.
+     * please refer to https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand
+     *
+     * it might be deprecated at some point.
+     *
+     * the newer method is the navigator.clipboard writeText API but only available on new browsers and doesn't support IE.
+     * please refer to https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText
+     *
+     * the navigator.clipboard specification is still on draft
+     * please refer to https://w3c.github.io/clipboard-apis/#dom-clipboard-writetext
+     */
+    const fakeElement = document.createElement('textarea');
+
+    fakeElement.value = self.event.xOpenpaasVideoconference;
+    fakeElement.setAttribute('style', 'height:0;width:0');
+    document.body.appendChild(fakeElement);
+    fakeElement.select();
+
+    try {
+      document.execCommand('copy');
```

Comment:
I answered you on the ticket for the solution. Please make sure to check it.
https://github.com/OpenPaaS-Suite/esn-frontend-calendar/issues/132#issuecomment-698675327

Reply:
thanks